docker/bin/stop-presto

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/bin/bash

set -euo pipefail
```

This makes bash behave more reasonably (more strict)

Reply:
I added `-u` (treat unset parameters as an error while substititing).
I'll skip `-e` and `-o pipeline` (exit immediately if a command exits with a non-zero status), because `curl` and `grep` would exit with error. We'll handle the error explicitly instead of terminate the script.